packages/nestjs-redis/package.json

```diff
@@ -0,0 +1,82 @@
+{
+  "name": "@voiceflow/nestjs-redis",
+  "description": "Redis for NestJS",
+  "version": "1.0.0",
+  "author": "Voiceflow",
+  "bugs": {
+    "url": "https://github.com/voiceflow/libs/issues"
+  },
+  "dependencies": {
+    "@voiceflow/nestjs-common": "^1.3.0"
```

Comment:
do we need to pin these inter-monorepo dependencies? i forget

Reply:
Yes, good call!